modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/SearchServiceDatabaseImpl.java

```diff
@@ -477,6 +479,18 @@ public Date getDeletionDate(String mediaPackageId) throws NotFoundException, Sea
     }
   }
 
+  public boolean exist(String mediaPackageId) throws SearchServiceDatabaseException {
```

Comment:
I'm being pedantic here, but this should be `exists`.  I would probably also rename it completely, although I'm not sure to what.  The mediapackage *does* exist, it's just deleted.  But naming this `notDeleted` isn't very usable either.  Could we flip the logic internally, and call it `deleted` or something?

Reply:
@gregorydlogan Thank you for the review! :)
Would `isAvailable()` be better? I will put Javadoc above it in any case (forgot that).

Reply:
Yeah actually, that's way better!